crates/typst-library/src/model/footnote.rs

```diff
@@ -15,30 +15,30 @@ use crate::model::{Destination, Numbering, NumberingPattern, ParElem};
 use crate::text::{SuperElem, TextElem, TextSize};
 use crate::visualize::{LineElem, Stroke};
 
-/// A footnote.
+/// 脚注。
 ///
-/// Includes additional remarks and references on the same page with footnotes.
-/// A footnote will insert a superscript number that links to the note at the
-/// bottom of the page. Notes are numbered sequentially throughout your document
-/// and can break across multiple pages.
+/// 脚注を用いて追加のコメントや参考文献を同じページに記述します。
+/// 脚注は、ページ下部の注釈にリンクする上付きの番号を挿入します。
+/// 注釈は文書全体で連続してナンバリングされ、
+/// 複数のページにまたがることができます。
 ///
-/// To customize the appearance of the entry in the footnote listing, see
-/// [`footnote.entry`]($footnote.entry). The footnote itself is realized as a
-/// normal superscript, so you can use a set rule on the [`super`] function to
-/// customize it. You can also apply a show rule to customize only the footnote
-/// marker (superscript number) in the running text.
+/// 脚注リストの項目の外観をカスタマイズするには、
+/// [`footnote.entry`]($footnote.entry)を参照してください。
+/// 脚注自体は通常の上付き文字として実現されているため、
+/// [`super`]関数に対してsetルールを適用してカスタマイズ出来ます。
```

Comment:
「出来る」は「できる」と表記するのが一般的だと思います。

cf. https://www.bunka.go.jp/seisaku/bunkashingikai/kokugo/hokoku/pdf/92895101_01.pdf

```suggestion
/// [`super`]関数に対してsetルールを適用してカスタマイズできます。
```